web/src/components/EcosystemSankey.vue

```diff
@@ -85,15 +85,18 @@ export default Vue.extend({
   computed: {
     sankeyOptions() {
       return {
-        height: 400,
+        // Make the chart height dependent on the number of nodes with a minimum of 500px
+        height: (this.sankeyData.length * 4 > 500 ? this.sankeyData.length * 4 : 500),
```

Comment:
```suggestion
        height: Math.max(this.sankeyData.length * 4, 500),
```

Reply:
changed in a2c8a83f6159cb98eaa3ff58adf83eaa8983a2c9